AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java

```diff
@@ -1027,6 +1027,17 @@ private void showStartupScreensAndDialogs(SharedPreferences preferences, int ski
                         startActivityForResultWithoutAnimation(infoIntent, SHOW_INFO_NEW_VERSION);
                     }
                 } else {
+                    // Spam for icon poll
+                    ConfirmationDialog dialog = new ConfirmationDialog ();
```

Comment:
Codacy would have flagged this though :-), I can't fix it since I'm not a repo admin here to set API keys and debug etc :man_shrugging: 
```suggestion
                    ConfirmationDialog dialog = new ConfirmationDialog();
```

Note that I don't think this is worth *actually* committing this change and having it in the changelog since it'll just revert anyway, and whitespace -> meh normally